packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -999,20 +994,14 @@ export abstract class FluidDataStoreContext
 				safeTelemetryProps,
 			);
 
-			sendGCUnexpectedUsageEvent(
-				this.mc,
+			this.mc.logger.sendTelemetryEvent(
 				{
 					eventName: "GC_Tombstone_DataStore_Changed",
-					category: this.gcThrowOnTombstoneUsage ? "error" : "generic",
-					gcTombstoneEnforcementAllowed: this.gcTombstoneEnforcementAllowed,
+					category: "generic",
 					callSite,
 				},
-				this.pkg,
```

Comment:
Weird, `Data_pkg` is empty for the one hit we have of this event, which is for `callSite: submitMessage`.  But to your point in the PR description, `Data_fullPackagePath` is set.

Reply:
Maybe it is from before we added pkg here? Not sure.